app/src/main/java/mollie/tictactoe/players/MobilePlayer.java

```diff
@@ -0,0 +1,29 @@
+package mollie.tictactoe.players;
+
+import ttt.Player;
+import ttt.game.Board;
+import ttt.game.Marks;
+
+public class MobilePlayer implements Player {
+
+    private final Marks mMark;
+
+    public MobilePlayer(Marks mark) {
+        mMark = mark;
+    }
+
+    @Override
+    public Marks getMark() {
+        return mMark;
+    }
+
+    @Override
+    public int getLocation(Board board) throws Exception {
```

Comment:
In the context of a player of tictactoe, I can't quite see what their "location" represents? Is it the last move they played?